src/v0/destinations/sfmc/transform.js

```diff
@@ -182,6 +183,26 @@ const responseBuilderForInsertData = (
   return response;
 };
 
+// DOC : https://developer.salesforce.com/docs/marketing/marketing-cloud/references/mc_rest_interaction/postEvent.html
+
+const responseBuilderForMessageEvent = (message, subDomain, authToken, hashMapEventDefinition) => {
```

Comment:
Are we exporting this for testing purposes ?

Reply:
Yes, for `transform.test.js`